zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerStats.java

```diff
@@ -41,6 +41,12 @@
 
     private final BufferStats clientResponseStats = new BufferStats();
 
+    //This will be a P0 alert
```

Comment:
I suggest we remove these comments because alerting and stats are separate things. Even we must say something here, I'd tend to make these suggestions instead of mandates (in which case I'd also clearly state the alert level - e.g. what P0 means). I would just remove these comments for clarity.

Reply:
sure. its needed onl for SFDC internally, not for the community, i'll remove the comments